runtime/j9vm/exports.cmake

```diff
@@ -446,6 +446,12 @@ else()
 	)
 endif()
 
+if(NOT JAVA_SPEC_VERSION LESS 23)
+	jvm_add_exports(jvm
+		JVM_GetCDSConfigStatus
+	)
+endif()
```

Comment:
Please also remove exports and implementations of these functions which are no longer needed in JDK23.
```c
JNIEXPORT jboolean JNICALL JVM_IsCDSDumpingEnabled(JNIEnv *);
JNIEXPORT jboolean JNICALL JVM_IsSharingEnabled(JNIEnv *);
JNIEXPORT jboolean JNICALL JVM_IsDumpingClassList(JNIEnv *);
```
UMA builds should be updated similarly.

Reply:
Disabled these three methods for JDK23+.